src/main/scala/ru/otus/sc/counter/model/CounterRequest.scala

```diff
@@ -0,0 +1,3 @@
+package ru.otus.sc.counter.model
+
+case class CounterRequest()
```

Comment:
```suggestion
case object CounterRequest
```

Класс без параметров это синглтон.